lighthouse-cli/chrome-launcher.ts

```diff
@@ -164,14 +164,14 @@ class ChromeLauncher {
         this.cleanup(client);
         reject(err);
       });
-      client.once('connect', _ => {
+      client.once('connect', (_: any) => {
```

Comment:
instead of this _ hack to omit () we should just use `() => {` as there are no arguments to this fn.